jest.config.jsdom.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
```

Comment:
Should `testEnvironment` still be `node`?

Reply:
yeah... its default, setup function will take care of the rest